docs/guides/admin/docs/workflowoperationhandlers/send-email-woh.md

```diff
@@ -28,16 +28,18 @@ Parameter Table
 |------------------|-------|-----------|-------------|
 |body|Email body content.<br>Takes precedence over body-template-file.|`<Recording Title> (<Mediapackage ID>)`|Lecture 1 (4bf316fc-ea78-4903-b00e-9976b0912e4d)|
 |body-template-file|Name of file that will be used as a template for the content of the email body.|EMPTY|templateName|
+|body-html|Email body content send as HTML.<br>Takes precedence over body-template-file-html.|EMPTY|`<h1>${mediapackage.identifier}</h1>`|
+|body-template-file-html|Name of file that will be used as a template for the HTML content of the email body.|EMPTY|templateNameHTML|
```

Comment:
What do you think of using `body-html-template-file` instead? For me, the `template-file` is a postfix to the first part, either `body` or `body-html`.

Reply:
For me it was the other way around, but I don't really care. Can do.

Reply:
Option is now renamed to your suggestion.